flink-core/src/test/java/org/apache/flink/core/fs/AbstractRecoverableWriterTest.java

```diff
@@ -196,16 +200,29 @@ private void testResumeAfterMultiplePersist(
         final Map<String, RecoverableWriter.ResumeRecoverable> recoverables = new HashMap<>(4);
         RecoverableFsDataOutputStream stream = null;
         try {
-            stream = initWriter.open(path);
-            recoverables.put(INIT_EMPTY_PERSIST, stream.persist());
-
-            stream.write(testData1.getBytes(StandardCharsets.UTF_8));
-
-            recoverables.put(INTERM_WITH_STATE_PERSIST, stream.persist());
-            recoverables.put(INTERM_WITH_NO_ADDITIONAL_STATE_PERSIST, stream.persist());
-
-            // and write some more data
-            stream.write(testData2.getBytes(StandardCharsets.UTF_8));
+            // This is just for locate  the root cause:
```

Comment:
nit: This is just for locate  the root cause: -> This is just to provide diagnostics to locate the root cause: